README.md

```diff
@@ -77,15 +77,21 @@ This installation guide is for ISIS users interested in installing ISIS (3.6.0)+
 
         conda config --env --add channels usgs-astrogeology
 
+        # If you have followed conda-forges advice to set `channel_priority=strict`,
+        # you need to set it to flexible for the install:
+        conda config --set channel_priority flexible
+
 
 6.  The environment is now ready to download ISIS and its dependencies:
 
-        conda install -c usgs-astrogeology isis
+        # To install e.g. version 4.3
+        conda install isis=4.3
 
     If you would like to work with our latest ISIS version 3, rather than updating
     to ISIS 4, instead run:
 
-	    conda install -c usgs-astrogeology isis=3.10.0
+	    # This will find the latest 3.x.y release
+        conda install isis=3
```

Comment:
I don't think this will work.  Did you test it?  I mean, it would if the conda isis package had been developed as a conda package for many years, and versions and such were coherent, but as far as I know you must do `conda install isis3` for the latest ISIS 3.* or `conda install isis3=3.6.1` to get a specific version.

Reply:
confirmed. that's what happpens (with `channel_priority=flexible`)